tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -217,58 +217,73 @@ public function test_are_site_fields_synced() {
 		$this->assertEquals( $failure_response_data, false );
 	}
 
+	/**
+	 * @runInSeparateProcess
+	 */
 	public function test_url_handles_staging() {
 		$url = $this->google_proxy->url();
 		$this->assertEquals( $url, Google_Proxy::PRODUCTION_BASE_URL );
-		// The test for this behaviour depends on a constant value which can
-		// only be redefined if PECL extension runkit/7 is installed.
-		if ( ! extension_loaded( 'runkit7' ) && ! extension_loaded( 'runkit' ) ) {
-			$this->markTestSkipped( 'The runkit7 or runkit extension is not available.' );
-		}
 
 		define( 'GOOGLESITEKIT_PROXY_URL', Google_Proxy::STAGING_BASE_URL );
+
 		$url = $this->google_proxy->url();
 		$this->assertEquals( $url, Google_Proxy::STAGING_BASE_URL );
-		if ( function_exists( 'runkit7_constant_remove' ) ) {
-			runkit7_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
-		} elseif ( function_exists( 'runkit_constant_remove' ) ) {
-			runkit_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
-		}
 	}
 
-	public function test_url_handles_development() {
+	/**
+	 * @runInSeparateProcess
+	 */
+	public function test_url_ignores_invalid_values() {
+		define( 'GOOGLESITEKIT_PROXY_URL', 'https://example.com' );
```

Comment:
There is some duplicate coverage here but this ensures the `url` method uses the new sanitize method which has more thorough coverage. I've also removed the need for the `runkit` extension here for manipulating constants at runtime which most don't have locally. It's quite slow to run in a separate process like this so I've kept these tests to a minimum.